lab_2/main.py

```diff
@@ -92,7 +291,36 @@ def find_diff_in_sentence(original_sentence_tokens: tuple, suspicious_sentence_t
     :param lcs: a longest common subsequence
     :return: a tuple with tuples of indexes
     """
-    pass
+    orig_indices = []
+    susp_indices = []
+    all_indices = []
+
+    for index, (original, suspect) in enumerate(zip(original_sentence_tokens, suspicious_sentence_tokens)):
+        if (original != suspect and original_sentence_tokens[index - 1: index] in lcs
+                and original_sentence_tokens[index: index + 1] in lcs):
+            orig_indices.append(index)
+            susp_indices.append(index)
+            orig_indices.append(index + 1)
+            susp_indices.append(index + 1)
+
+        elif (original != suspect and tuple(original_sentence_tokens[index + 1:]) != suspect):  # original_sentence_tokens[index + 1:] not in lcs
+            orig_indices.append(index)
```

Comment:
why do you check `original_sentence_tokens[index + 1:]` but add `index` to indices?